CONTRIBUTORS.md

```diff
@@ -1723,3 +1723,4 @@ Symfony is the result of the work of many people who made the code better
  - Thomas BERTRAND (sevrahk)
  - Matej Žilák (teo_sk)
  - Vladislav Vlastovskiy (vlastv)
+ - Ole Rößner (basster)
```

Comment:
As far as I know, this file is auto-generated, so doesn't need to be manually modified